.fossa.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019-present Open Networking Foundation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
from now on, we can use the shorter header, can remove this part.

Reply:
Done